rs/crypto/internal/crypto_lib/bls12_381/type/src/lib.rs

```diff
@@ -1647,7 +1657,7 @@ macro_rules! declare_mul2_table_impl {
 
                     // This is the only difference from the constant time version:
```

Comment:
This comment is no longer up to date as there is also the initialization of the accumulator above.

Reply:
:+1: updated